composer.json

```diff
@@ -53,5 +56,10 @@
     },
     "conflict": {
         "lolautruche/ez-core-extra-bundle": "<2.0"
+    },
+    "config": {
+        "allow-plugins": {
+            "composer/package-versions-deprecated": true
```

Comment:
probably required for all LTS-supported versions

Reply:
Yep, I will be going through all v2 repos when migrating them to GA - will be added then